src/actions/guides/database/database_setup.cr

```diff
@@ -41,18 +82,38 @@ class Guides::Database::DatabaseSetup < GuideAction
     * retry_attempts
     * retry_delay
 
-    To set the connection pool options, just append a query string to the end of the
-    `settings.url` option (e.g. `?initial_pool_size=5&retry_attempts=2`)
+    To set the connection pool options, just set the `query` option in your `Avram::Credentials` to a query string.
+    (e.g. `initial_pool_size=5&max_pool_size=10`).
 
-    ### Avram Configuration
+    > If using an environment variable, the query string should come at the end of your connection string.
+    > (e.g. `postg)
```

Comment:
I'm not sure I understand this example. Maybe a full database url example would help?

Reply:
😂 I must have been in the middle of writing it and got side tracked. Oops!